Web/Edubase.Web.UI/Views/Guidance/EstablishmentBulkUpdate.cshtml

```diff
@@ -32,840 +32,837 @@
-                    <td class="govuk-table__cell" data-label="Database name (use this for your upload file)">ChairOfProprietorBodyCounty</td>
-                    <td class="govuk-table__cell" data-label="Equivalent field name in app">Chair of Proprietor's Body County:</td>
-                </tr>
-                <tr class="govuk-table__row">
-                    <td class="govuk-table__cell" data-label="Database name (use this for your upload file)">ChairOfProprietorBodyPostcode</td>
-                    <td class="govuk-table__cell" data-label="Equivalent field name in app">Chair of Proprietor's Body Postcode:</td>
-                </tr>
+<h2 class="govuk-heading-m">Appendix</h2>
+<table class="govuk-table gias-table gias-table">
+<thead class="govuk-table__head">
+<tr class="govuk-table__row">
+    <th class="govuk-table__header">
+        Database name (use this for your upload file)
+    </th>
+    <th class="govuk-table__header">
+        Equivalent field name in app
+    </th>
+</tr>
+</thead>
+<tbody class="govuk-table__body">
+<tr class="govuk-table__row">
+    <td class="govuk-table__cell" data-label="Database name (use this for your upload file)">EstablishmentName</td>
+    <td class="govuk-table__cell" data-label="Equivalent field name in app">School / College Name</td>
+</tr>
+<tr class="govuk-table__row">
+    <td class="govuk-table__cell" data-label="Database name (use this for your upload file)">Street</td>
+    <td class="govuk-table__cell" data-label="Equivalent field name in app">Street</td>
+</tr>
+<tr class="govuk-table__row">
+    <td class="govuk-table__cell" data-label="Database name (use this for your upload file)">Locality</td>
+    <td class="govuk-table__cell" data-label="Equivalent field name in app">Locality</td>
+</tr>
+<tr class="govuk-table__row">
+    <td class="govuk-table__cell" data-label="Database name (use this for your upload file)">Address3</td>
+    <td class="govuk-table__cell" data-label="Equivalent field name in app">Address3</td>
+</tr>
+<tr class="govuk-table__row">
+    <td class="govuk-table__cell" data-label="Database name (use this for your upload file)">Town</td>
+    <td class="govuk-table__cell" data-label="Equivalent field name in app">Town</td>
+</tr>
+<tr class="govuk-table__row">
+    <td class="govuk-table__cell" data-label="Database name (use this for your upload file)">County (name)</td>
+    <td class="govuk-table__cell" data-label="Equivalent field name in app">County (name)</td>
+</tr>
+<tr class="govuk-table__row">
+    <td class="govuk-table__cell" data-label="Database name (use this for your upload file)">Postcode</td>
+    <td class="govuk-table__cell" data-label="Equivalent field name in app">Postcode</td>
+</tr>
+<tr class="govuk-table__row">
+    <td class="govuk-table__cell" data-label="Database name (use this for your upload file)">LA</td>
+    <td class="govuk-table__cell" data-label="Equivalent field name in app">Local authority</td>
+</tr>
```

Comment:
It is now ambiguous which value should be provided (name or ID or code).

Reading through the bug comments I _think_ it's the LA code which is expected.

Either way, elaborating here would be helpful I think.

Reply:
Discussed with @RogerHowellDfE 
This is ambiguous as the value updates the LA using either Code or Name.
There are much bigger questions about the guidance as updating LA also updates other fields